packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -752,7 +752,7 @@ class ScriptPopupMenu extends StatelessWidget {
                 Text('Copy filename',
                     style: Theme.of(context).regularTextStyle),
                 const Icon(
-                  Icons.copy,
+                  Icons.file_copy,
```

Comment:
can you use `Icons.content_copy` instead? this will match what we already use in the debugger console.